examples/rasta_grpc_bridge/cpp/main.cpp

```diff
@@ -106,30 +110,27 @@ void processConnection(std::function<std::thread()> run_thread) {
     terminator_event.carry_data = s_connection;
     terminator_event.fd = s_terminator_fd[0];
     enable_fd_event(&terminator_event);
-    add_fd_event(&s_rc->rasta_lib_event_system, &terminator_event, EV_READABLE);
+    rasta_add_fd_event(s_rc, &terminator_event, EV_READABLE);
 
     // Forward gRPC messages to rasta
     auto forwarderThread = run_thread();
-    
-    char* buf = new char[recv_msg_size];
+
+    char *buf = new char[recv_msg_size];
     int recvlen;
     while ((recvlen = rasta_recv(s_rc, s_connection, buf, recv_msg_size)) > 0) {
         static std::mutex s_busy_writing;
         std::lock_guard<std::mutex> guard(s_busy_writing);
 
         std::lock_guard<std::mutex> streamGuard(s_busy);
         if (s_currentStream != nullptr) {
-            logger_log(s_rc->h.logger, LOG_LEVEL_DEBUG, (char *)"RaSTA retrieve", (char *)"forwarding packet to grpc");
```

Comment:
Log output might still be helpful for debugging.

Reply:
Agreed. Since I don't want to use the librasta-internal logging here, we could either go by using printf/iostream (no verbosity configurable without further effort) or include a simple logging library. This would be enough to justify another PR, I guess...